python/hsfs/core/feature_group_engine.py

```diff
@@ -24,7 +24,9 @@ class FeatureGroupEngine(feature_group_base_engine.FeatureGroupBaseEngine):
     APPEND = "append"
 
     def __init__(self, feature_store_id):
-        feature_group_base_engine.FeatureGroupBaseEngine.__init__(feature_store_id)
+        feature_group_base_engine.FeatureGroupBaseEngine.__init__(
+            self, feature_store_id
+        )
```

Comment:
okay, I don't understand this, that's where you should use `super` - or better, since you are not doing anything in the init than what is done in the base class' init, I believe you can omit the `__init__` alltogether, so it will just have the one of the base class. Adding the init in a childclass just overwrites the base class one, that's why you call `super().__init__()` but in this case you don't even need that.

![image](https://user-images.githubusercontent.com/8422705/98654507-b2d24000-233e-11eb-9762-4d55f4b0f34b.png)